spec/routing/users_routing_spec.rb

```diff
@@ -11,6 +11,10 @@
       expect(get: "/users/1/edit").to route_to("users#edit", id: "1")
     end
 
+    it "routes to #edit with guest account" do
```

Comment:
@hectorcorrea do you need to put the `.com` back onto the id in the edit?

Reply:
The controller did that already, so once the route was handled correctly (i.e. call `edit()` in the controller) the code takes care of putting the id back together.

But the issue in this case was that Rails was failing to even *recognize the route* if it came with the `.com` as part of the `:id`. 

What I still don't know is why Rails did recognize this kind of route for the Show page? We had issues in the Show page but those were in the controller (i.e. Rails recognized the route and called the proper controller method).